Demos/Blazorise.Demo.Bootstrap.RC/Pages/_Host.cshtml

```diff
@@ -18,7 +18,7 @@
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
     <link rel="stylesheet" href="https://unpkg.com/easymde/dist/easymde.min.css">
 
-    <link href="_content/Blazorise/blazorise.css?v=0.9.4.1" rel="stylesheet" />
+    <link href="_content/Blazorise/blazorise.min.css.gz?v=0.9.4.1" rel="stylesheet" />
```

Comment:
Don't use minified files in demo apps. It becomes harder to debug any JS code.

Reply:
You're right. It was kinda curiosity getting the best of me. I was experimenting with gzip compressed resources 🤣🤣 I hadn't used them before.

Reply:
I'll fix and finish the remaining projects later today then.

Reply:
No problem :)